### PR TITLE
Thread user roles delete

### PR DESCRIPTION
**Pull request by lawalkeyd**
Created the views in the users app. Added the necessary URLs, serializers and views. Think other contributors can build on this

### CODE REVIEW

**Comment by charles-co** [comment]
resolve conflicts

**Comment by lawalkeyd** [comment]
Resolved

File: backend/.flake8 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [flake8]
 max-line-length = 120
 exclude = .tox,.git,*/migrations/*,*/static/CACHE/*,docs,node_modules
-extend-ignore = F405,F403,W503,W504
+extend-ignore = F405,F403,W503,W504,E501
```
Comment (charles-co) [inline backend/.flake8:4]: hmm did this work out?

**Comment by charles-co** [comment]
> Created the views in the users app. Added the necessary URLs, serializers and views. Think other contributors can build on this

we'll discuss more, good work

**Comment by charles-co** [comment]
ready for PR review by mark

**Pull request merged**